examples/js/ShaderSkin.js

```diff
@@ -167,7 +167,7 @@ THREE.ShaderSkin = {
 				"diffuseColor = diffuseColor * colDiffuse;",
 
 				"vec3 normal = normalize( vNormal );",
-				"vec3 viewPosition = normalize( vViewPosition );",
+				"vec3 viewerDirection = normalize( vViewPosition );",
```

Comment:
We have used `viewDir` and `viewVector` elsewhere in the library for this term. It looks like the latter one would most consistent with the source code in this PR.